### DISCUSSION TITLE
The Zoobank number of the taxonomic act has which subsubsection attribute?

### DISCUSSION THREAD

**Discussion by Carol-Sokolowicz** [plazi/community#82] [Q&A] [answered]
The zoobank number of the species description act is place right after the species name.
It is included as a treatment right? Under what subsubsection? Description?

![image](https://user-images.githubusercontent.com/57491712/73843090-b1e30f80-47fc-11ea-8675-a64692b04a25.png)

Thanks!

**Comment by myrmoteras** [comment, marked as answer]
subSubSection type=nomenclature

**Comment by Carol-Sokolowicz** [comment]
Ok! Thanks Donat.
